packages/nimble-components/src/utilities/tests/rich-text-markdown-string.ts

```diff
@@ -1 +1,2 @@
-export const richTextMarkdownString = 'Supported rich text formatting options:\n1. **Bold**\n2. *Italics*\n3. Numbered lists\n   1. Option 1\n   2. Option 2\n4. Bulleted lists\n   * Option 1\n   * Option 2\n5. Absolute link: <https://nimble.ni.dev/>\n 6. @mention:\n    1. User pattern: <user:1>\n    2. HTTPS pattern: <https://user/2>';
+export const richTextMarkdownString = '<user:1>';
```

Comment:
Looks like some temporary testing code snuck in here.

Reply:
woops, that was me too 😅 but yea can be cleaned up

Reply:
Updated